README.md

```diff
@@ -1,14 +1,46 @@
+
+
+(Note - include the icons for the stack + team members)
+
+--
+
 # Chuck Norris Joke API
 
+## Overview
 A simple project created to practice working in a team environment using Agile methodologies.
 
-(Note - include the icons for the stack + team members)
+**Live link:** [https://chingu-voyages.github.io/v42-toucans-team-02/](https://chingu-voyages.github.io/v42-toucans-team-02/)
+
+
+## Features
+* Clicking nav bar items drop user to appropriate page elements
+* Searching for a joke consumes the Chuck Norris API and returns an object
+* Color schemas abide by contrast ratios for accessibility requirements
+* etc...
+
+
+## Running The Project
+Live verson:
+[Chuck Norris Jokes](https://chingu-voyages.github.io/v42-toucans-team-02/)
+
+From the repo:
+
+  1. Clone this project locally
+  2. ...
+  3. ...
+  4. Invite friends or family over and tell some jokes!
+
 
-Here is Gillian's Readme comment for testing:
-Hello team!
+## Tech Used / Dependencies
+* Vanilla JavaScript
+* Add-on packages:  
+    * ...
+* Color inspiration from: ...
 
```

Comment:
Here is the code for the images for your convenience:

```html
    <h3 align="left">Languages and Tools:</h3>
    <p align="left">
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg"
        alt="html5"
        width="40"
        height="40"
      />
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg"
        alt="css3"
        width="40"
        height="40"
      />
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/javascript/javascript-original.svg"
        alt="javascript"
        width="40"
        height="40"
      />
      <img
        src="https://www.vectorlogo.zone/logos/git-scm/git-scm-icon.svg"
        alt="git"
        width="40"
        height="40"
      />
    </p>
```

Reply:
Added